src/utils.js

```diff
@@ -1,12 +1,22 @@
-export function objectWithKey (key, value) {
-  return ((Array.isArray(value) && value.length > 0) || (!Array.isArray(value) && value)) ? {[key]: value} : {}
+import { config } from '@fortawesome/fontawesome-svg-core'
+import semver from 'semver'
+
+export const ICON_PACKS_STARTING_VERSION = '7.0.0-alpha1'
+
+const svgCorePackageJson = require('@fortawesome/fontawesome-svg-core/package.json')
+
+export const SVG_CORE_VERSION = svgCorePackageJson.version
+
+export function objectWithKey(key, value) {
+  return (Array.isArray(value) && value.length > 0) || (!Array.isArray(value) && value) ? { [key]: value } : {}
```

Comment:
function from above, it is just moved down due to the imports and other new `const`'s.

I made no changes to this function.